languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -8193,18 +8193,41 @@ though, and has since been slightly modified)-->
       </rule>
     </rulegroup>
     <rulegroup id="ACCORD_NOMBRE" name="Accord en nombre">
+    <antipattern>
+        <token regexp="yes">\d+</token>
+        <token regexp="yes">heures?</token>
+        <token regexp="yes">\d+</token>
+    </antipattern>
+    <antipattern>
+        <token regexp="yes">formule|saisons?|épisodes?|zones?</token>
+        <token regexp="yes">\d+</token>
+    </antipattern>
+    <antipattern>
+        <token regexp="yes">\d+</token>
+        <token regexp="yes">éditions</token>
+    </antipattern>
+    <antipattern>
+        <token regexp="yes">Windows|Apple|Galaxy</token>
+        <token regexp="yes">\d+</token>
+        <token postag=".*s"/>
+    </antipattern>
```

Comment:
I suggest to add examples.
They will help to understand why we have antipattern and the will be useful for regression testing.

Reply:
Understood, I'll remember to add examples in the future.